SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -97,7 +97,14 @@ void SpatialDispatcher::ProcessOps(const TArray<Worker_Op>& Ops)
 			break;
 
 		case WORKER_OP_TYPE_FLAG_UPDATE:
-			SpatialWorkerFlags->ApplyWorkerFlagUpdate(Op.op.flag_update);
+			if (Op.op.flag_update.value == nullptr)
+			{
+				SpatialWorkerFlags->RemoveWorkerFlag(Op.op.flag_update.name);
+			}
+			else
+			{
+				SpatialWorkerFlags->SetWorkerFlag(Op.op.flag_update.name, Op.op.flag_update.value);
```

Comment:
These need to be converted to FStrings explicitly vai UTF8_TO_TCHAR, although I think this will currently do the right thing as it happens.

Reply:
+1